metricbeat/helper/module.go

```diff
@@ -162,13 +162,18 @@ func (m *Module) FetchMetricSets(b *beat.Beat, metricSet *MetricSet) {
 	events, err := metricSet.Fetch()
 
 	if err != nil {
-		// TODO: Also list module?
-		logp.Err("Fetching events in MetricSet %s returned error: %s", metricSet.Name, err)
-		// TODO: Still publish event with error
-		return
-	}
+		errorMessage := fmt.Sprintf("Fetching events for MetricSet %s in Module %s returned error: %s", metricSet.Name, m.name, err)
```

Comment:
Why not remove the `Sprintf` and just call `logp.Err()` directly with the same arguments?